hack/make/build-deb

```diff
@@ -96,7 +96,7 @@ set -e
 			;;
 		esac
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
-			RUN dpkg-buildpackage -uc -us
+			RUN dpkg-buildpackage -uc -us -I.git
```

Comment:
that makes more sense lol, looks good to me :)

On Fri, Apr 22, 2016 at 10:40 AM, Ken Cochrane <notifications@github.com>
wrote:

> In hack/make/build-deb
> <https://github.com/docker/docker/pull/22236#discussion_r60775513>:
>
> > @@ -96,7 +96,7 @@ set -e
> >  			;;
> >  		esac
> >  		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
> > -			RUN dpkg-buildpackage -uc -us
> > +			RUN dpkg-buildpackage -uc -us -I.git
>
> I believe it stands for Ignore. but I could be wrong :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/22236/files/820fd002836ffbd5e16942213fe69ab149c44a9e#r60775513>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>